aiven/client/client.py

```diff
@@ -349,6 +349,43 @@ def add_service_kafka_acl(self, project, service, permission, topic, username):
     def delete_service_kafka_acl(self, project, service, acl_id):
         return self.verify(self.delete, "/project/{}/service/{}/acl/{}".format(project, service, acl_id))
 
+    def get_available_kafka_connectors(self, project, service):
+        return self.verify(self.get, "/project/{}/service/{}/available-connectors".format(project, service))
+
+    def list_kafka_connectors(self, project, service):
+        return self.verify(self.get, "/project/{}/service/{}/connectors".format(project, service))
+
+    def get_kafka_connector_status(self, project, service, connector_name):
+        return self.verify(self.get, "/project/{}/service/{}/connectors/{}/status".format(project, service, connector_name))
+
+    def get_kafka_connector_configuration(self, project, service, connector_name):
+        url = "/project/{}/service/{}/connector-plugins/{}/configuration"
+        return self.verify(self.get, url.format(project, service, connector_name))
+
+    def create_kafka_connector(self, project, service, config):
+        return self.verify(self.post, "/project/{}/service/{}/connectors".format(project, service), body=config)
+
+    def update_kafka_connector(self, project, service, connector_name, config):
+        url = "/project/{}/service/{}/connectors/{}"
+        return self.verify(self.put, url.format(project, service, connector_name), body=config)
+
+    def delete_kafka_connector(self, project, service, connector_name):
+        return self.verify(self.delete, "/project/{}/service/{}/connectors/{}".format(project, service, connector_name))
```

Comment:
URL encoding in existing code is handled badly so this is not worse than existing code as such but connector names are known to be able to contain any characters so need to have `connector_name` => `quote(connector_name, safe="")`